RELEASE.md

```diff
@@ -1,3 +1,83 @@
+# Release 1.15.0
+
+## Major Features and Improvements
+
+## Breaking Changes
+
+## Bug Fixes and Other Changes
+
+* Promoting `unbatch` from experimental to core API.
+* Adds `tf.enable_control_flow_v2()` and `tf.disable_control_flow_v2()`.
+* EagerTensor now support buffer interface for tensors.
+* This change bumps the version number of the FullyConnected Op to 5.
+* tensorflow : crash when pointer become nullptr.
+* `tf.keras.estimator.model_to_estimator` now supports exporting to tf.train.Checkpoint format, which allows the saved checkpoints to be compatible with `model.load_weights`.
+* Add support for defaulting the value of `cycle_length` argument of `tf.data.Dataset.interleave` to the number of schedulable CPU cores.
+* parallel_for: Add converter for `MatrixDiag`.
+* Saving a Keras Model using `tf.saved_model.save` now saves the list of variables, trainable variables, regularization losses, and the call function.
+* Add `narrow_range` attribute to `QuantizeAndDequantizeV2` and V3.
+* `OMP_NUM_THREADS` is no longer used by the default Keras config. To configure the number of threads, use `tf.config.threading` APIs.
+* Added new op: `tf.strings.unsorted_segment_join`.
+* `tf.keras.model.save_model` and `model.save` now defaults to saving a TensorFlow SavedModel. A SavedModel code is only applicable to `tf.keras`.
```

Comment:
I don't understand the last sentence in this bullet.

Should this be in breaking changes?

Reply:
Added to breaking changes, and removed the last line.